new-frontiers-14/frontier-station-14
# More descriptive ship sale & cryo pod messages

## About the PR


Adds more descriptive error messages when trying to sell docked ships and enter a cryo pod.  If there are multiple organics aboard, the error message will name the first one that is onboard.

## Why / Balance


Hunting for something you don't know is annoying.  Trying to arrange a Crescent sale when somebody's gone SSD or something's gained consciousness without your knowledge is a pain.

## How to test


1. Start a server, join on two clients.
2. Buy a ship on client 1.
3. Walk onto the ship on client 2.
4. Try to sell the ship on client 1, it should tell you that client 2's character is aboard.
5. Move the ship on client 2.
6. Try to sell the ship on client 1, it should tell you that the ship is undocked.
7. On client 2, dock the ship, spawn a pAI, start looking for AI.
8. On client 2, /ghost and take the pAI.
9. On client 1, pick up the pAI, run to the cryo pod.
10. Try to enter the cryo pod, the pod should tell you about the pAI in your inventory.

## Media


![crew must disembark](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/24c4978e-8f3b-4dfd-8e94-7d48eb8ec223)
![sus amogus](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/2fa0b86b-0a6e-4f4f-9b4e-db12f4a59912)
![must be docked](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/0d0e7c41-b94c-436c-bdc3-18c44d3e5d55)
![syndie_ai_cryo](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/d05f4260-21e7-4864-bda5-e64172a7c034)


## Breaking changes


**Changelog**



- tweak: When selling a ship, the console will tell you who is onboard, or if the ship is undocked.
